calcom/cal.com
# Separate and paginate team query for members

## What does this PR do?



Since Vercel only allows 4MB max for API payloads, this PR separates the "viewer.teams.get" call into one that gets the team info and another get gets the team members. The get team members query is also paginated so we should avoid the 4MB limit that Vercel imposes. 

Fixes # (issue)


https://www.loom.com/share/12ee4dc070d94bb28adb363944994014

**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Sign into the teampro account
- In settings go to the Seeded team page
    - Click load more memebers. If there are only 6 members then the button should disappear
    - Invite a team member
    - Delete a member
    - Change the role of a member
- Go to the team's profile page
    - Make changes and save
- Go to the teams appearance page
    - Disable branding and save

## Checklist


- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
